File: Cocoa Programming/Cocoa Pr L69 ObjC/Cocoa Pr L69 ObjC/MainWindowController.m
//
//  MainWindowController.m
//  Cocoa Pr L69 ObjC
//
//  Created by Michele Galvagno on 02/09/22.
//

#import "MainWindowController.h"
#import "ViewController.h"

@interface MainWindowController ()

@end

@implementation MainWindowController
- (instancetype)init
{
    self = [super initWithWindowNibName:@"MainWindowController" owner:self];
    
    return self;
}

- (void)windowDidLoad {
    [super windowDidLoad];
    
    [self setContentViewController:[[ViewController alloc] init]];
}

@end

File: Cocoa Programming/Cocoa Pr L69 NSWindowController/Cocoa Pr L69 NSWindowController/MainWindowController.swift
//
//  MainWindowController.swift
//  Cocoa Pr L69 NSWindowController
//
//  Created by Michele Galvagno on 28/08/22.
//

import Cocoa

class MainWindowController: NSWindowController {
    convenience init() {
        self.init(windowNibName: "MainWindowController")
    }
//    override var windowNibName: NSNib.Name? {
//        "MainWindowController"
//    }

    override func windowDidLoad() {
        super.windowDidLoad()

        // Implement this method to handle any initialization after your window controller's window has been loaded from its nib file.
        contentViewController = ViewController()
    }
    
}
